srcpkgs/makeself/template

```diff
@@ -1,31 +1,19 @@
 # Template file for 'makeself'
 pkgname=makeself
 version=2.4.0
-revision=2
+revision=3
 archs=noarch
-create_wrksrc=yes
+wrksrc="${pkgname}-release-${version}"
 makedepends="tar gzip bzip2"
```

Comment:
Is this necessary?
Most (all?) of our packages require `tar(1)` and either `gzip(1)` or `bzip2(1)` or `xz(1)` to extract.
And we don't need to specify them as `makedepends`.
`xbps-install(1)` also requires them to run but `srcpkgs/xbps/template` doesn't have them.

Anyway, if those packages should be included here, how about `xz(1)` and other compression package? And package for cryptographic options?

---
Sorry, but I would like to have conscious from the team.

Reply:
I don't know actually, I've made minimal changes and left other stuff as it was before.

Reply:
@congdanhqx: Good point! Just one detail missed: tar is always installed as `hostmakedepends`.
@innerspacepilot: Thanks for contribution!
As you are modifying template, remove makedepends as they are not needed and depends as these packages are required by base system and please replace `$pkgname` with `makeself` in all template.
If anything non-standard (like xz) is needed at runtime, you can add it to depends.

Reply:
done